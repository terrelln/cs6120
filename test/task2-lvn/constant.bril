@main() {
    x: int = const 0;
    y: int = const 1;
    z: int = add x y;
    print z;
}