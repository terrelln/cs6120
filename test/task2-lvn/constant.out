@main {
.__block0:
  y: int = const 1;
  print y;
}
